seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*[\\s]+style\\s*?=\\s*('|\\\")?\\s*([\\s\\S]*;+|)*\\s*color\\s*?:\\s*?white\\s*;*\\s*('|\\\")?/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
The file passed validation so I thought it was ok. I will change to properly escape the single quote.

The intent was to allow any preliminary style attributes as a way of being more flexible such as (style="color:green;font-weight:bold;color:white;") which does render as white text so it should still pass.  Also I did say it does not handle every case of potential format issues such as, it doesn't handle typos of extra < >, mismatched quotes (single vs double), etc. but if the results render on their screen I wouldn't expect to have to check for those. Is this still or would you prefer that I investigate a non-regex solution?